src/styles/main.scss

```diff
@@ -0,0 +1,103 @@
+$day-size: 100px;
+$transition-time: 0.5s;
+$calendar-width: 7 * ($day-size + 1px);
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$calendar-padding: 20px;
+$bg-color: #eee;
+
+body {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-counter;
+  flex-wrap: wrap;
+  display: flex;
+  width: $calendar-width;
+  padding: $calendar-padding;
+  gap: 1px;
+
+  &__day {
```

Comment:
use the rules for structuring your SCSS that a past mentor gave you, that's really good advice